### ISSUE TITLE
Release Binary Assets CI is failling

### ISSUE THREAD

**Issue by SergioGasquez** [cargo-generate/cargo-generate#657]
Workflow `release-binary-assets.yml` is failing for all the targets with:
```
Error: Found argument '-V' which wasn't expected, or isn't valid in this context
```
I tried looking at what could be causing this issue but I haven't found any changes

**Comment by sassman** [comment]
this is going to be fixed with the `v0.15.1` that is about to go out

**Comment by SergioGasquez** [comment]
Great, thanks!

**Comment by sassman** [comment]
fixed with `v0.15.2`
